CMakeLists.txt

```diff
@@ -161,7 +161,9 @@ add_compile_definitions(MUMBLE_TARGET_OS="${MUMBLE_TARGET_OS}")
 
 set(CMAKE_UNITY_BUILD_BATCH_SIZE 40)
 
-add_subdirectory(src)
+if(client OR server)
```

Comment:
If you want to use these options here, they have to be declared in this file as well (and not as currently in `src/CMakeLists.txt`). Otherwise, they will always be false, if not set explicitly, because cmake has not yet processed the option's default values.